HISTORY.md

```diff
@@ -0,0 +1,13 @@
+# Changelog
```

Comment:
If you don't need to, don't add `HISTORY.md` please.

Reply:
In this case, changelog generator wants to make retroactive changes otherwise.

I'll double check if the `# Changelog` heading is actually expected in `HISTORY.md` though